packages/ui/src/PasswordField.tsx

```diff
@@ -25,6 +25,7 @@ type PasswordFieldCoreProps = {
   value?: string
   onBlur?: (e: FocusEvent<HTMLInputElement>) => void
   onChange: (e: ChangeEvent<HTMLInputElement>) => void
+  withIcon?: boolean
```

Comment:
Should it be in `PasswordFieldExtraProps`? Otherwise, we won;t be able to pass it in from the `PasswordFieldFormik`

Reply:
Indeed, I've updated it. Thanks :)